CustomDynamicFont/Resources/Assets/Assets.xcassets/star.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "star.pdf",
+      "idiom" : "universal"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
One small optimization point that we don't have to create new assets here and we can just use `UIImage(systemName: "imageName")` directly since there are star icons in the system images already 🙏

![Screen Shot 2021-12-02 at 10 29 20](https://user-images.githubusercontent.com/70877098/144353032-e0687db4-5366-4b82-82ab-557780874c96.png)

A reference can be found [here](https://stackoverflow.com/questions/23409610/how-to-use-default-ios-images/56440003) => Example usage for our app can simply be 😁
```
// Declaration
enum SystemImageName: String {
    case star
    case starFill = "star.fill"
}
...

// Usage
tabBarItem.image = UIImage(systemName: SystemImageName.star.rawValue)
tabBarItem.selectedImage = UIImage(systemName: SystemImageName.starFill.rawValue)
```

Reply:
This project supports >=iOS 11 and `UIImage(systemImage:)` is supported from iOS13.0+. I decided to import the star image to reduce the complexity of `if#available` in this project.

Reply:
thanks for the insights @blyscuit 👍